fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
Also, should the existing `kickerText` function (to be renamed `kickerHTML`) not return `Some(...)` for the data-less kickers like `BreakingNewsKicker`? The plain-text function I want would do this; would keeping the behaviours of the two functions different be a good idea, or would there be a case for making them consistent?